usr.bin/minimal_count_stats/minimal_count_stats.c

```diff
@@ -1,12 +1,15 @@
 /*-
- * Copyright (c) 2016 SRI International
- * Copyright (c) 2016 Robert N. M. Watson
- * All rights reserved.
```

Comment:
You lost this

Reply:
Yeah that should probably never have been part of the file, I wrote the file from scratch in 2019...

Reply:
If you feel strongly about this, I can fix it, but I'd prefer not to.